docs/source/en/using-diffusers/kerascv.mdx

```diff
@@ -0,0 +1,171 @@
+
+* [Teach StableDiffusion new concepts via Textual Inversion](https://keras.io/examples/generative/fine_tune_via_textual_inversion/)
+* [Fine-tuning Stable Diffusion](https://keras.io/examples/generative/finetune_stable_diffusion/)
+* [DreamBooth](https://keras.io/examples/generative/dreambooth/)
+* [Prompt-to-Prompt editing](https://github.com/miguelCalado/prompt-to-prompt-tensorflow)
+
+A Stable Diffusion model has three primary components:
+
+* Text encoder 
+* UNet 
+* VAE 
+
+Depending on the fine-tuning task, we may fine-tune one of more of these components (the VAE is almost always left untouched). Here are some common combinations:
+
+* DreamBooth: Text encoder and UNet 
+* Classical text2image: UNet 
+* Textual Inversion: Text encoder 
+
+Let's use [this checkpoint](https://huggingface.co/sayakpaul/textual-inversion-kerasio/resolve/main/textual_inversion_kerasio.h5) which was generated
+by conducting Textual Inversion with the following "placeholder token": `<my-funny-cat-token>`. 
+
+On the tool, we supply the following things: 
+
+* Path(s) to download the fine-tuned checkpoint(s) (KerasCV)
+* An HF token 
+* Placeholder token (only applicable for Textual Inversion)
+
+<div align="center">
+<img src="https://i.imgur.com/JMATVKJ.png"/>
+</div> 
+
+As soon as you hit "Submit", the conversion process will begin. Once it's complete, you should see the following:
+
+<div align="center">
+<img src="https://i.imgur.com/gJyVD9g.png"/>
+</div> 
+
+If you click the [link](https://huggingface.co/sayakpaul/textual-inversion-cat-kerascv_sd_diffusers_pipeline/tree/main), you
+should see something like so: 
+
+<div align="center">
+<img src="https://i.imgur.com/vl6w7Zm.png"/>
+</div> 
+
+If you head over to the [model card of the repository](https://huggingface.co/sayakpaul/textual-inversion-cat-kerascv_sd_diffusers_pipeline), the
+following should appear: 
+
+<div align="center">
+<img src="https://i.imgur.com/PxCNRFh.png"/>
+</div> 
+
+_**Note that we're not specifying the UNet weights here since the UNet is not fine-tuned during Textual Inversion.**_
+
+And that's it! You now have your fine-tuned KerasCV Stable Diffusion model in Diffusers 🧨
+
+## Usage in Diffusers 
+
+Just besides the model card of the [repository](https://huggingface.co/sayakpaul/textual-inversion-cat-kerascv_sd_diffusers_pipeline), 
+you'd notice an inference widget to try out the model directly from the UI 🤗
+
+<div align="center">
+<img src="https://i.imgur.com/jv8Ikes.png"/>
+</div> 
+
+On the top right hand side, we provide a "Use in Diffusers" button. If you click the button, you should see the following code-snippet: 
+
+```py 
+from diffusers import DiffusionPipeline
+
+pipeline = DiffusionPipeline.from_pretrained("sayakpaul/textual-inversion-cat-kerascv_sd_diffusers_pipeline")
+```
+
+Let's perform inference! 
```

Comment:
```suggestion
The model is in standard `diffusers` format. Let's perform inference!
```

(Just to stress it, so they don't need to study the following code in depth if they are already familiar with the API).